playbooks/templates/.env.j2

```diff
@@ -4,9 +4,9 @@
 #
 # REQUIRED: server_domain and portal_domain need to be defined by user
 # 
-# NOTE: | alternates are for handling backwards compatibility. 
-{% set server_domain = ansible_host | webportal_server_config.domain_name %}
-{% set portal_domain = portal_domain | webportal_server_config.portal_cluster_domain %}
+# NOTE: `| default(...)` alternates are for handling backwards compatibility.
+{% set server_domain = ansible_host %}
```

Comment:
`ansible_host` is an Ansible spacial variable and is defined always, the `| default(...)` code is never executed for it.